gpu-miner.c

```diff
@@ -100,9 +109,11 @@ double grindNonces(size_t items_per_iter, int cycles_per_iter) {
 		if (ret != CL_SUCCESS) { printf("failed to write to targmobj buffer: %d\n", ret); exit(1); }
 
 		// Execute OpenCL kernel as data parallel
+		// Note that minimum intensity is 8, making the local
+		// worksize always divisible by the global worksize
```

Comment:
Yes; it was really late that night - I did an off-by-one in my head. You're correct; 1 << 7 == 0x80 or 128.